.travis.yml

```diff
@@ -1,19 +1,12 @@
 language: go
 go:
-  - release
-  - tip
+  - stable
+  - master
 sudo: false
-before_install:
-  - gotools=golang.org/x/tools
-  - if [ "$TRAVIS_GO_VERSION" = "release" ] || [ "$TRAVIS_GO_VERSION" = "go1.3.3" ]; then gotools=code.google.com/p/go.tools; fi
 install:
   - go get -d -t -v ./...
-  - go get -v $gotools/cmd/cover
-  - go get -v $gotools/cmd/vet
-  - go get -v github.com/bradfitz/goimports
-  - go get -v github.com/golang/lint/golint
+  - go get -v github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
Could we stick with using the tools independently rather than with golangci-lint?